odl/solvers/advanced/proximal_operators.py

```diff
@@ -966,11 +1060,44 @@ def proximal_l1(space, lam=1, g=None, isotropic=False):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The :math:`L_1`-norm :math:`F` is the functional
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_1.
+
+    For a step size :math:`\\sigma`, the proximal operator of :math:`\\sigma F`
+    is
+
+        .. math::
+
+            prox[\\sigma F](y) = \\left\{ \\begin{array}{ll}
+            y - \\sigma \\lambda
+            & \\quad  \\text{if } y > g + \\sigma \\lambda, \\\\
+            0 &\\quad
+            g - \\sigma \\lambda \\leq y \\leq g +
+            \\sigma \\lambda \\\\
+            y + \\sigma \\lambda
+            & \\quad  \\text{if } y < g - \\sigma \\lambda,
+            \\end{array} \\right.
```

Comment:
This is a bit confusing since the inequalities in the cases have to read pointwise (I guess), i.e. per point or alternatively per vector entry.